cmd/deploy.go

```diff
@@ -353,8 +353,8 @@ func updateVersion(apiClient client.AuroraConfigClient, applications []string, v
 		return err
 	}
 
-	// Save config file
-	if err := apiClient.PutAuroraConfigFile(auroraConfigFile, eTag); err != nil {
+	// Save config file (Gobo)
+	if _, err = DefaultAPIClient.UpdateAuroraConfigFile(auroraConfigFile, eTag); err != nil {
 		return err
```

Comment:
Jeg ser at err ganske konsekvent returneres uten å legge på ytterligere context. Er dette et mønster dere har blitt enige om er ok, eller er det tilfeldig? Personlig mener jeg å returnere err uten context i mange tilfeller ikke bør gjøres. Det kan ofte bli veldig vanskelig å finne ut av hvor en feil oppstod uten mer context.

Reply:
Verdt å lese siden dere er på Go 1.13: https://blog.golang.org/go1.13-errors

Reply:
Takk for tilbakemelding.  Jeg skal se nøyere på dette med feilhåndtering. Mulig det er såpass omfattende i ao at vi kan ta det som en egen Jirasak, så blir det ikke en flekkvis fiks.

Reply:
Generelt bør det være en målsetting at feilmeldingene inneholder tilstrekkelig kontekst. For ao vil kontektsten i mange tilfeller være gitt av kommandoen som ble brukt. Men vi bør nok ta en gjennomgang og undersøke om vi skal berike flere av feilene som kan oppstå med mer kontekst.